numba_cuda/numba/cuda/cudadrv/driver.py

```diff
@@ -82,25 +73,43 @@
     "to be available"
 )
 
+
 # Enable pynvjitlink if the environment variables NUMBA_CUDA_ENABLE_PYNVJITLINK
 # or CUDA_ENABLE_PYNVJITLINK are set, or if the pynvjitlink module is found. If
 # explicitly disabled, do not use pynvjitlink, even if present in the env.
-_pynvjitlink_enabled_in_env = _readenv(
-    "NUMBA_CUDA_ENABLE_PYNVJITLINK", bool, None
+def _get_option(name, fallback):
+    env_val = _readenv("NUMBA_" + name, bool, None)
+    if env_val is not None:
+        return env_val
+    cfg_val = getattr(config, name, None)
+    if cfg_val is not None:
+        return cfg_val
+    return fallback
+
+
+ENABLE_PYNVJITLINK = _get_option(
+    "CUDA_ENABLE_PYNVJITLINK",
+    fallback=importlib.util.find_spec("pynvjitlink") is not None,
 )
-_pynvjitlink_enabled_in_cfg = getattr(config, "CUDA_ENABLE_PYNVJITLINK", None)
-
-if _pynvjitlink_enabled_in_env is not None:
-    ENABLE_PYNVJITLINK = _pynvjitlink_enabled_in_env
-elif _pynvjitlink_enabled_in_cfg is not None:
-    ENABLE_PYNVJITLINK = _pynvjitlink_enabled_in_cfg
-else:
-    ENABLE_PYNVJITLINK = importlib.util.find_spec("pynvjitlink") is not None
-
 if not hasattr(config, "CUDA_ENABLE_PYNVJITLINK"):
     config.CUDA_ENABLE_PYNVJITLINK = ENABLE_PYNVJITLINK
 
 
+USE_NV_BINDING = _get_option("CUDA_USE_NVIDIA_BINDING", fallback=False)
```

Comment:
If the fallback is `False`, doesn't this make the NVIDIA binding disabled by default?